### PR TITLE
Service binder update

### PR DESCRIPTION
**Pull request by ramirezg**
### Bound services have credentials available to view, but only when use requests them.
screen shot 2015-08-13 at 2 52 16 pm

### users are warned before viewing data
screen shot 2015-08-13 at 3 01 13 pm

### users are warned before unbinding service
screen shot 2015-08-13 at 3 01 23 pm

### users are told the app will need to be restaged before the service is available.
screen shot 2015-08-13 at 3 01 29 pm

### CODE REVIEW

**Comment by ramirezg** [comment]
@jcscottiii just finished testing for this, thoughts?

File: static/app/views/app.html [hunk 1]
```diff
@@ -169,10 +171,26 @@ <h3 class="panel-title">{{service.entity.name}}</h3>
...
+                ng-show="service.boundService" mwl-confirm title="Warning!" message="Are you sure you want to unbind this service?"
+                on-confirm="unbindService(service)" confirm-button-type="danger"
+                cancel-button-type="default">Unbind Service</button>
+            </div>
+            <div class="panel-heading" ng-show='service.credentials'>
+                <h3 class="panel-title">VCAP Info</h3>
+            </div>
+            <div class="panel-body" ng-show='service.credentials' ng-init='showCredentials=false'>
+                <button type="button" class="btn btn-xs btn-danger" ng-hide="showCredentials" mwl-confirm
+                title="Warning!" message="These are senstive data DO NOT SHARE!"
```
Comment (jcscottiii) [inline static/app/views/app.html]: Let's be more specific with the message.
"This info may contain sensitive data such as passwords. DO NOT SHARE!"

I kind of want to say something about watching your surroundings before clicking but not sure how to say it concisely.

**Comment by jcscottiii** [comment]
Needs to be rebased. But looks good minus the message.

**Comment by jcscottiii** [comment]
Rebase again.

**Pull request merged**